src/isolate.rs

```diff
@@ -330,4 +372,49 @@ mod tests {
     let op = Box::new(futures::future::ok(control));
     (true, op)
   }
+
+  #[test]
+  fn test_metrics() {
+    let argv = vec![String::from("./deno"), String::from("hello.js")];
+    let mut isolate = Isolate::new(argv, metrics_dispatch);
+    tokio_util::init(|| {
+      // verify that metrics have been properly initialized
+      {
+        let g = isolate.state.metrics.lock().unwrap();
+        let metrics = g.deref();
+        assert_eq!(metrics.ops_executed, 0);
+        assert_eq!(metrics.bytes_sent, 0);
+        assert_eq!(metrics.bytes_recv, 0);
+      }
+
+      isolate
+        .execute(
+          "y.js",
+          r#"
+          const control = new Uint8Array([4, 5, 6]);
+          const data = new Uint8Array([42, 43, 44, 45, 46]);
+          libdeno.send(control, data);
+        "#,
+        )
+        .expect("execute error");
+      isolate.event_loop();
+      let g = isolate.state.metrics.lock().unwrap();
+      let metrics = g.deref();
+      assert_eq!(metrics.ops_executed, 1);
+      assert_eq!(metrics.bytes_recv, 5);
+      assert_eq!(metrics.bytes_sent, 4);
```

Comment:
Reverse your usage of "sent" and "recv" - it should be from the perspective of the javascript user.